theme/js/script.js

```diff
@@ -40,13 +40,16 @@ $(document).ready(function() {
 
     postCardComponentJsonData();
 
+    slideToggleAccordionContent();
+
+    addActiveClassToAccordion();
```

Comment:
Can delete this function after moving the code.

Reply:
done